curator/utils.py

```diff
@@ -894,6 +894,8 @@ def get_client(**kwargs):
             )
     try:
         client = elasticsearch.Elasticsearch(**kwargs)
+        if 'api_key' in kwargs:
```

Comment:
Perhaps this will work better if it is explicit:

```
if 'api_key' in kwargs and 'api_key' is not None:
```
Because a `None` value for `api_key` might mean it's still present in `kwargs`, this explicitly prevents a `None` value from triggering the client header update.

Reply:
Yes, should work much better, have changed it as you suggested.